services/web/project/static/swagger.yml

```
@@ -0,0 +1,275 @@
+        title:
+          type: string
+          description: The contact's job title
+          
+        email:
+          type: string
+          description: The contact's email address.
+          
+        mobile:
+          type: integer
+          description: The contact's mobile phone number.
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)